Model/Cron/Customers.php

```diff
@@ -101,7 +104,7 @@ public function getList($last_update)
                     'email' => $subscriberData['email'],
                     'name' => $subscriberData['name'],
                     'store' => $this->storeManager->getStore($s['store_id'])->getName(),
-                    'website_id' => $subscriberData['website_id']
+                    'website' => $this->websiteRepository->getById($websiteId)->getName()
```

Comment:
Why not use `$this->storeManager->getWebsite(...)` method? Will it yield a different result?

Reply:
That would return application website object
https://www.magentoextensions.org/documentation/class_magento_1_1_store_1_1_model_1_1_store_manager.html#ad2374a35b412219cd541bd74f241a476

Reply:
> That would return application website object
> https://www.magentoextensions.org/documentation/class_magento_1_1_store_1_1_model_1_1_store_manager.html#ad2374a35b412219cd541bd74f241a476

I looked into the documentation of website object, it has a function `getName()` which we can use. Previously I tried `getWebsiteName() `which wasn't a real function so I went that path using `websiteRepository`. 
Added in commit